src/integ_test_resources/ios/sdk/integration/cdk/cdk_integration_tests_ios/iot_stack.py

```diff
@@ -205,7 +213,78 @@ def setup_custom_authorizer(self):
             },
         )
 
-    def setup_custom_authorizer_function(self) -> str:
+    def setup_custom_authorizerUserPass(self):
+        custom_authorizer_name = "iot_custom_authorizerUserPass"
+        self._parameters_to_save["custom_authorizerUserPass_name"] = custom_authorizer_name
+        token_key_name = "IoTTokenKeyName"
+        self._parameters_to_save["custom_authorizerUserPass_token_key_name"] = token_key_name
+        token_value = "allow"
+        self._parameters_to_save["custom_authorizerUserPass_token_value"] = token_value
+
+        iot_custom_authorizer_key_resource = self.create_custom_authorizer_signing_key_generic("2",
+            "Manages an asymmetric CMK and token signature for iot custom authorizer with username and password.",
+            token_value)
+
+        custom_authorizer_token_signature = iot_custom_authorizer_key_resource.get_att(
+            "custom_authorizer_token_signature"
+        ).to_string()
+        self._parameters_to_save[
+            "custom_authorizerUserPass_token_signature"
+        ] = custom_authorizer_token_signature
+
+        #Force region to 'us-east-1' due to enhanced custom authorizers only available in this region
+        authorizer_function_arn = self.setup_custom_authorizer_function("2",
+                                                                        "custom_resources/iot_custom_authorizerUserPass_function",
+                                                                        "iot_custom_authorizerUserPass.handler",
+                                                                        "Sample custom authorizer that allows or denies based on username and password",
+                                                                        "us-east-1")
+        create_authorizer_policy = aws_iam.PolicyStatement(
+            effect=aws_iam.Effect.ALLOW, actions=["iot:CreateAuthorizer",
+                                                  "iot:UpdateAuthorizer",
+                                                  "iot:DeleteAuthorizer",
+                                                  "iot:UpdateDomainConfiguration",
+                                                  "iot:CreateDomainConfiguration",
+                                                  "iot:DescribeDomainConfiguration",
+                                                  "iot:DeleteDomainConfiguration"], resources=["*"],
+        )
+        provider_lambda = aws_lambda.SingletonFunction(
+            self,
+            "iot_custom_authorizerUserPass_provider_lambda",
+            uuid="iot_custom_authorizerUserPass_provider_lambda_20200507150213",
```

Comment:
This number looks oddly specific. Is it just one that you came up with randomly? Or does it have meaning/importance to some externally-managed resource?

Reply:
It was a copy and paste of number that was used for the original `iot_custom_authorizer_provider_lambda`.  I'm not exactly sure where the number came from.. hmm..